### PR TITLE
Documentation: improve permissions docs

### PR DESCRIPTION
**Pull request by shamoon**
## Proposed change

It is something else to maintain but hopefully useful. LMK if I missed anything of course.

## Type of change

- [ ] Bug fix: non-breaking change which fixes an issue.
- [ ] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [x] Documentation only.
- [ ] Other. Please explain:

## Checklist:

- [ ] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -206,50 +206,77 @@ for details.
...
 ### Default permissions
 
-Default permissions for documents can be set using workflows.
+[Workflows](#workflows) provide advanced ways to control permissions.
 
 For objects created via the web UI (tags, doc types, etc.) the default is to set the current user
-as owner and no extra permissions, but you explicitly set these under Settings > Permissions.
+as owner and no extra permissions, but you can explicitly set these under Settings > Permissions.
+
+Documents consumed via the consumption directory or directly to the API do not have an owner or
```
Comment (stumpylog) [inline docs/usage.md:231]: Doesn't the API take the request user as the owner?

https://github.com/paperless-ngx/paperless-ngx/blob/025001499d27723f83d43dc75ff2f4705b2369b8/src/documents/views.py#L957

File: docs/usage.md [hunk 2]
```diff
@@ -206,50 +206,77 @@ for details.
...
+| ShareLink     | Grants global permissions to add, delete or view Share Links.                                                                                                                                       |
+| StoragePath   | Grants global permissions to add, edit, delete or view Storage Paths.                                                                                                                               |
+| Tag           | Grants global permissions to add, edit, delete or view Tags.                                                                                                                                        |
+| UISettings    | Grants global permissions to add, edit, delete or view the UI settings that are used by the web app.<br/>Users expected to access the web UI should usually be granted at least _View_ permissions. |
+| User          | Grants global permissions to add, edit, delete or view Groups.                                                                                                                                      |
```
Comment (stumpylog) [inline docs/usage.md:269]: This says Groups, but should be Users

**Review by stumpylog — CHANGES_REQUESTED** [review]
Thanks for the cleanup, couple suggestions

**Comment by shamoon** [comment]
Thanks stumpy, great catch x2. Let me know if you notice anything else

**Review by stumpylog — APPROVED** [review]

**Pull request merged**